apicast/conf.d/apicast.conf

```diff
@@ -8,9 +8,13 @@ set_by_lua $deployment 'return os.getenv("THREESCALE_DEPLOYMENT_ENV");';
 location = /threescale_authrep {
   internal;
 
+  set_by_lua_block $log {
```

Comment:
Because this is just used here. The others are set on the parent call and copied to the post action. But this is just to have some tiny bit of logic (not adding `&` if there is no content) in this http call.